### DISCUSSION TITLE
Firmware version v20221002 - REV D - Cannot find CPU AUTO function

### DISCUSSION THREAD

**Discussion by ScottMcPhee69** [UzixLS/zx-sizif-512#41] [General]
Hi,

One of the features I really liked about the latest firmware version v20221002 is the ability to use the "AUTO" CPU speed function, this will greatly speed up browsing for games and loading.

We burned the firmware but we cannot locate the AUTO CPU speed setting, the CPU speed settings are as before with no AUTO option.

We are using Revision "D" firmware.

Can anyone help please?

**Comment by UzixLS** [comment]
It's inside advanced submenu
![изображение](https://github.com/UzixLS/zx-sizif-512/assets/1556813/d5ed7782-2e99-4664-a4d7-5f28fc913dfd)

**Comment by ScottMcPhee69** [comment]
Hi Eugene 

We have applied the firmware but the advanced submenu still looks the same as it did. 

We are running revision D boards. 

Is there any way to check the firmware version from the SIZIF-512 itself? 

Scott




Get TypeApp for Android

**Comment by ScottMcPhee69** [comment]
Attached is a picture of the flashing showing the verification. 

Could it be a bug in the revision D version of the firmware? 

Scott




Get TypeApp for Android

**Comment by UzixLS** [comment]
https://github.com/UzixLS/zx-sizif-512/issues/40
please dont spam to all places. thank you :)
